### PR TITLE
ACM-2111 - Addon status column

### PR DESCRIPTION
**Pull request by Randy424**
regarding: https://issues.redhat.com/browse/ACM-2111

### CODE REVIEW

**Comment by Randy424** [comment]
Screenshot 2023-02-10 at 3 49 22 PM

**Comment by Randy424** [comment]
/hold
Requires unit testing

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/components/useClusters.tsx [hunk 1]
```diff
@@ -101,12 +107,14 @@ export function useClusters(
   const groupManagedClusterInfos = managedClusterInfos.filter((mci) => clusterNames.includes(mci.metadata.namespace))
 
   const groupHostedClusters = hostedClusters.filter((hc) => clusterNames.includes(hc.metadata?.name))
-
+  console.log('mapClusters')
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ClusterSets/components/useClusters.tsx:110]: Stray console log.

**Comment by Randy424** [comment]
FYI, will close this PR in favor of [a new one.](https://github.com/stolostron/console/pull/2568)

**Pull request closed without merging**